optimum/commands/export/openvino.py

```diff
@@ -491,6 +492,10 @@ def run(self):
                 from optimum.intel import OVModelForCausalLM
 
                 model_cls = OVModelForCausalLM
+            elif task.startswith("text2text-generation"):
+                from optimum.intel import OVModelForSeq2SeqLM
+
+                model_cls = OVModelForSeq2SeqLM
```

Comment:
why do we need it?

Reply:
This is our approach to apply data-aware quantization during export from CLI. This differs from data-free approach applied during export by the fact that in order to collect quantization calibration data, we require a created instance of the model class. In this case this is an instance of `OVModelForSeq2SeqLM`.